model.go

```diff
@@ -237,3 +237,44 @@ type Language struct {
 	Status string `json:"status"`
 	Name   string `json:"name"`
 }
+
+// UserIDPage represents a single page of UserIDs returned by the "following or "followers" endpoint.
+type UserIDPage struct {
```

Comment:
There's already a type that matches this signature called [IDs](https://github.com/crowdriff/twitter/blob/master/model.go#L257). I think it'd be cleaner to use one type for this kind of response.